tests/dragonfly/replication_test.py

```diff
@@ -1919,3 +1919,41 @@ async def test_heartbeat_eviction_propagation(df_local_factory):
     keys_replica = await c_replica.execute_command("keys *")
     assert set(keys_master) == set(keys_replica)
     await disconnect_clients(c_master, *[c_replica])
+
+
+@pytest.mark.asyncio
+async def test_policy_based_eviction_propagation(df_local_factory, df_seeder_factory):
```

Comment:
We have this test above, I asked Yue on his PR to add it and mark it as skip. You can remove remove his test.
Note that there is unsupported_types in seeder in his test because another bug that Kostas fixed

Reply:
it messes up the diff, but the change is really only adding the test below